docs/DeveloperGuide.md

```diff
@@ -595,20 +626,71 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
 
    1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No person is deleted. Error details shown in the status message.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Editing a person
+
+1. Editing a person in the displayed list of persons.
+
+    1. Prerequisites: List all persons using the `list` command. At least one person in the list.
+
+    1. Test case: `edit 1 n/John Tan`<br>
+       Expected: First person in the list has their name edited to `John Tan`. Details of the edited person is shown in the status message.
+
+    1. Other correct edit commands to try: `edit 1 p/12345678`, `edit 1 d/Accounting` and `edit 1 r/Manager`<br>
+       Expected: Similar to previous, the respective fields gets edited.
+
+    1. Test case: `edit 0`<br>
+       Expected: No person is edited. Error details shown in the status message.
+
+    1. Other incorrect delete commands to try: `edit`, `edit x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.       
+
+### Sorting the list
+
+1. Sorting the displayed list of persons.
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+
+    1. Test case: `sort n/`<br>
+       Expected: Displayed list is sorted by alphabetical order.
+
+    1. Other correct sort commands to try: `sort d/`, `sort r/` and `sort e/`<br>
+       Expected: Similar to previous, the respective fields gets sorted based on lexicographical order.
+
+    1. Test case: `sort`<br>
+       Expected: Displayed list is not sorted. Error details shown in the status message.
+
+### Finding a person
+
+1. Finds a person by the given field.
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
```

Comment:
I think that this prerequisite is not necessary as `find` works even without any employees displayed in the displayed list.

Reply:
Hmmm while that is true, the prerequisite of having multiple persons listed ensures that the `find` command functionality can be tested. If the list is empty, executing a `find` command would result in the same output, an empty list, regardless of the command's correctness. But, with a populated list, we can determine if the command actually works so I think we should keep it. However, I understand that it is not completely true, maybe I can change it to `Prerequisite for effective test` so it is clearer?

Reply:
Okay I will!